pulsar-websocket/src/main/java/org/apache/pulsar/websocket/service/WebSocketProxyConfiguration.java

```diff
@@ -175,6 +176,63 @@ public class WebSocketProxyConfiguration implements PulsarConfiguration {
     @FieldContext(doc = "TLS cert refresh duration (in seconds). 0 means checking every new connection.")
     private long tlsCertRefreshCheckDurationSec = 300;
 
+    /**** --- KeyStore TLS config variables. --- ****/
+    @FieldContext(
+            doc = "Enable TLS with KeyStore type configuration in broker"
```

Comment:
```suggestion
            doc = "Enable TLS with KeyStore type configuration in web service"
```

Reply:
Good catch! I replaced it with `WebSocket` instead.